charts/pega/values.yaml

```diff
@@ -227,3 +227,7 @@ hazelcast:
   enabled: false
   # No. of initial members to join
   replicas: 3
+  # UserName to be used in client server hazelcast mode for authentication
+  username: ""
+  # Password to be used in client server hazelcast mode for authentication
+  password: ""
```

Comment:
Can we have these fields only in sub-chart . We will just try to keep minimal params in Values.yaml file. 
Also in the comment , we need to add like these settings are applicable from 8.6+ versions

Reply:
As discussed, since these fields are mandatory for hazelcast(without which the deployment fails), we have included them in the main chart's values.yaml